sdk/keyvault/azure-security-keyvault/azure/security/keyvault/keys/README.md

```diff
@@ -0,0 +1,255 @@
+# Azure Key Vault Key client library for Python
+Azure Key Vault allows you to create and store keys in the Key Vault. Azure Key Vault client supports RSA keys and elliptic curve keys, each with corresponding support in hardware security modules (HSM).
+
+ Multiple keys, and multiple versions of the same key, can be kept in the Key Vault. Cryptographic keys in Key Vault are represented as [JSON Web Key [JWK]](https://tools.ietf.org/html/rfc7517) objects. This library offers operations to create, retrieve, update, delete, purge, backup, restore and list the keys and its versions.
+
+[Source code](/azure/security/keyvault/keys) | [Package (PyPI)](TODO) | [API reference documentation](TODO) | [Product documentation](https://docs.microsoft.com/en-us/azure/key-vault/) | [Samples](/azure/security/keyvault/keys/samples)
+## Getting started
+### Install the package
+Install the Azure Key Vault client library for Python with [pip](https://pypi.org/project/pip/):
+
+```Bash
+pip install azure-security-keyvault
+```
+
+### Prerequisites
+* An [Azure subscription](https://azure.microsoft.com/free/).
+* Python 2.7, 3.4 or later to use this package.
+* An existing Key Vault. If you need to create a Key Vault, you can use the [Azure Cloud Shell](https://shell.azure.com/bash) to create one with this Azure CLI command. Replace `<your-resource-group-name>` and `<your-key-vault-name>` with your own, unique names:
+
+    ```Bash
+    az keyvault create --resource-group <your-resource-group-name> --name <your-key-vault-name>
+    ```
+
+### Authenticate the client
+In order to interact with the Key Vault service, you'll need to create an instance of the [KeyClient](TODO-rst-docs) class. You would need a **vault url** and **client secret credentials (client id, client key, tenant id)** to instantiate a client object for using the `DefaultAzureCredential` examples in the README. `DefaultAzureCredential` way of authentication by providing client secret credentials is being used in this getting started section but you can find more ways to authenticate with [azure-identity](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/identity/azure-identity).
```

Comment:
~client key~ client secret ("key" suggests crypto but the secret is an arbitrary string)